ctapipe/reco/hillas_intersection.py

```diff
@@ -248,8 +246,8 @@ def _predict(self, hillas_dict, array_pointing, telescopes_pointings=None):
             az=sky_pos.altaz.az.to(u.rad),
             core_x=grd.x,
             core_y=grd.y,
-            core_tilted_x=core_x,
-            core_tilted_y=core_y,
+            core_tilted_x=u.Quantity(core_x, u.m),
```

Comment:
I think the problem is that `reconstruct_tilted` only returns floats and not Quantities. This was already taken into account for `core_tilted_uncert_{x,y} `, which is converted to a Quantity. 
If I remember correctly this resulted in some test failing for the `ProcessorTool`. It was not noticed before because the tests only included `HillasReconstructor` and not `HillasIntersection`

Reply:
Ah ok, so the units were just missing before, great then